packages/ui/src/components/va-date-picker/components/VaDatePickerCell.vue

```diff
@@ -92,101 +89,79 @@ export default defineComponent({
     opacity: 0;
   }
 
-  &_highlighted-weekend { color: var(--va-date-picker-weekends-color); }
-
-  &_selected {
-    background-color: var(--va-date-picker-selected-background);
-    color: var(--va-date-picker-selected-text);
+  &_highlighted-weekend {
+    color: var(--va-date-picker-weekends-color);
   }
 
+  // &_in-range:not(.va-date-picker-cell_disabled) {
+  // &_in-range { // should be like a the line above if the `range` value
+  // will exclude not allows dates
   &_in-range {
-    &::before {
-      border: 2px solid var(--va-date-picker-color);
-      opacity: 0.5;
+    color: var(--va-date-picker-color);
+
+    &::after {
+      background-color: var(--va-date-picker-selected-background);
+      opacity: var(--va-date-picker-cell-background-opacity-hover);
     }
   }
 
   &_today {
-    color: var(--va-date-picker-today-text);
+    color: var(--va-date-picker-color);
+    font-weight: bold;
 
-    &::after {
-      background-color: var(--va-date-picker-today-background);
-      opacity: var(--va-date-picker-today-background-opacity);
+    &::before {
+      border: 2px solid var(--va-date-picker-color);
     }
   }
 
-  &_selected.va-date-picker-cell_today {
-    &::before {
-      border: 2px solid var(--va-date-picker-selected-text);
-      opacity: 1;
-    }
+  &_selected {
+    background-color: var(--va-date-picker-selected-background);
 
-    &::after {
-      opacity: 1;
+    &,
+    &.va-date-picker-cell_today {
+      color: var(--va-date-picker-selected-text);
     }
   }
 
   &_other-month {
-    opacity: 0.5;
+    opacity: var(--va-date-picker-cell-opacity);
   }
 
   &:hover,
   &_focused,
   &_focused:hover {
-    $focus-border-opacity: 0.8;
-
-    &::before {
-      border: 2px solid var(--va-date-picker-focused-border-color);
-      opacity: $focus-border-opacity;
-    }
+    &:not(
+    .va-date-picker-cell_selected):not(
+      .va-date-picker-cell_readonly):not(
+        .va-date-picker-cell_disabled):not(
+          .va-date-picker-cell_other-month) {
```

Comment:
```suggestion
    &:not(.va-date-picker-cell_selected)
    :not(.va-date-picker-cell_readonly)
    :not(.va-date-picker-cell_disabled)
    :not(.va-date-picker-cell_other-month) {
```

Reply:
I checked this syntax but it doesn't work